src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "scss/variables/_colors";
+@import "scss/variables/_mixins";
+* {
+  font-family: "Roboto";
+}
+
+.container {
+  @include flex(space-around, center, column);
+  border-radius: 10px;
+  text-align: center;
+  display: inline-flex;
+  margin: 20px;
+  padding: 25px;
+  background-color: $primary;
+}
+
+.text {
+  color: $white;
+  font-size: 15px;
+  line-height: 18px;
+  white-space: pre;
```

Comment:
`pre` i think doesn't exist... verify this pls

Reply:
![pre](https://user-images.githubusercontent.com/78367161/120248885-4b82cb80-c24f-11eb-81eb-6033d663c799.png)